### ISSUE TITLE
php7 (ubuntu 16.04) doens't like rutorrent

### ISSUE THREAD

**Issue by Amdingo** [xombiemp/ultimate-torrent-setup#31]
Unless I'm missing something...

`[21.10.2016 03:08:13] WebUI started.
[21.10.2016 03:08:13] Bad response from server: (200 [parsererror,getplugins]) SyntaxError: Unexpected token <
[21.10.2016 03:08:13] Bad response from server: (200 [parsererror,getuisettings]) <?php require_once( 'util.php' ); $s = '{}'; $fname = getSettingsPath()."/uisettings.json"; $fo = fopen($fname, 'r'); if($fo!==false) { if(flock($fo, LOCK_SH)) { $s = @file_get_contents($fname); if($s==false) $s = '{}'; flock($fo, LOCK_UN); } fclose($fo); } cachedEcho($s,"application/json",true);`

**Comment by xombiemp** [comment]
You're missing something.  Did you follow the guide? Are you using the latest version of rutorrent?

**Comment by Amdingo** [comment]
I did.  And I can confirm that it was PHP 7, because when I downgraded to 5.6 it works fine.  No changes, just did `sudo a2dismod php7.0 ; sudo a2enmod php5.6 ; sudo service apache2 restart`

Kudos though, the work you put into this guide is amazing!  I'm working on a slightly more involved front-end to manage all of these as I also include sabnzbd in my installs.  Thought that adding rutorrent would be handy!
